src/term/mod.rs

```diff
@@ -1109,18 +1187,31 @@ impl ansi::Handler for Term {
         debug_println!("unset mode::APP_KEYPAD");
         self.mode.remove(mode::APP_KEYPAD);
     }
+
+    #[inline]
+    fn designate_charset(&mut self, index: CharsetIndex, charset: StandardCharset) {
```

Comment:
On the one hand I think 'configure' will instantly make more sense to most programmers, on the other 'designate' is more accurate and sticks to the specification. If we were rigorously implementing the entire specification I would go with designate, but otherwise changing it to 'configure' and documenting it seems more helpful.